lazyflow/utility/io_util/OMEZarrStore.py

```diff
@@ -0,0 +1,180 @@
+###############################################################################
+#   ilastik: interactive learning and segmentation toolkit
+#
+#       Copyright (C) 2011-2024, the ilastik developers
+#                                <team@ilastik.org>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# In addition, as a special exception, the copyright holders of
+# ilastik give you permission to combine ilastik with applets,
+# workflows and plugins which are not covered under the GNU
+# General Public License.
+#
+# See the LICENSE file for details. License information is also available
+# on the ilastik web site at:
+# 		   http://ilastik.org/license.html
+###############################################################################
+import json
+import logging
+import os
+from collections import OrderedDict
+from typing import Dict
+from urllib.parse import unquote_to_bytes
+
+import jsonschema
+import vigra
+from zarr.core import Array as ZarrArray
+from zarr.storage import FSStore, LRUStoreCache
+
+from lazyflow import rtype
+from lazyflow.utility import Timer
+from lazyflow.utility.io_util.multiscaleStore import MultiscaleStore, DEFAULT_SCALE_KEY
+
+logger = logging.getLogger(__name__)
+
+OME_ZARR_V_0_4_KWARGS = dict(dimension_separator="/", normalize_keys=False)
+OME_ZARR_V_0_1_KWARGS = dict(dimension_separator=".")
+
+
+def _get_axistags_from_spec(ome_spec: Dict) -> vigra.AxisTags:
+    if "axes" not in ome_spec:
```

Comment:
if you are covering both cases of the `if` statement, couldn't you phrase them to not use negative logic? That is

```python
if "axes" in ome_spec:  # instead of if NOT
    # positive case
else:
    # negative case
```

Reply:
This happened because I implemented the branches incrementally as I traced the changes in spec between versions. I like the fact that this ends up ordering the function logic by spec version. But I'm perfectly happy inverting the order by inverting both if-branchings 👍